GoogleTestAdapter/VsPackage.GTA/VsPackage.GTA.csproj

```diff
@@ -101,7 +101,7 @@
     <EmbeddedResource Include="Resources\ReleaseNotes\0.5.0.md" />
     <EmbeddedResource Include="Resources\ReleaseNotes\0.5.1.md" />
     <EmbeddedResource Include="Resources\ReleaseNotes\0.1.0.md" />
-    <EmbeddedResource Include="Resources\ReleaseNotes\0.2.0.md" />
+    <EmbeddedResource Include="Resources\ReleaseNotes\0.4.1.md" />
```

Comment:
Runaway find-and-replace :)

Reply:
Good catch! Especially since that built without issues.